WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -2,6 +2,37 @@ import UIKit
 import SafariServices
 import WordPressKit
 
+/// The source for the sign in flow for external tracking.
+public enum SignInSource {
+    /// Initiated from the WP.com login CTA.
+    case wpCom
+    /// Initiated from the WP.com login flow that starts with site address.
+    case wpComSiteAddress
+    /// Other unspecified sources.
+    case other
```

Comment:
Nit: I'm thinking this case is not really necessary, since we want to be as specific as possible in `enum`s. If we want to avoid breaking changes, when integrating this library we could use `@unknown default` case to handle new cases in the future. WDYT?

Reply:
Yea, I think it makes sense to not include this default case and only errors with specific sources are passed to the client app. I made an update in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/652/commits/8591d33e66b2baf7551b671b4c9adb36443a73fd